packages/cdn-location/data-generation/final-data/airportCodeToRegionNumber.ts

```diff
@@ -1,345 +1,345 @@
-    COK: 17500,
-    PAT: 17600,
-    KNU: 17700,
-    DEL: 17800,
-    IXC: 17900,
-    JSR: 18000,
-    CGP: 18100,
-    DAC: 18200,
-    PBH: 18300,
-    KTM: 18400,
-    LHE: 18500,
-    ISB: 18600,
-    KHI: 18700,
-    TAS: 18800,
-    ALA: 18900,
-    ULN: 19000,
-    ICN: 19100,
-    TYO: 19200,
-    NRT: 19300,
-    HND: 19400,
-    ITM: 19500,
-    KIX: 19600,
-    FUK: 19700,
-    OKA: 19800,
-    GUM: 19900,
-    CGY: 20000,
-    CEB: 20100,
-    MNL: 20200,
-    BWN: 20300,
-    SGN: 20400,
-    HAN: 20500,
-    PNH: 20600,
-    VTE: 20700,
-    MFM: 20800,
-    HKG: 20900,
-    KHH: 21000,
-    TPE: 21100,
-    FOC: 21200,
-    HYN: 21300,
-    HGH: 21400,
-    SHA: 21500,
-    CZX: 21600,
-    WHU: 21700,
-    KHN: 21800,
-    CGD: 21900,
-    KWE: 22000,
-    KMG: 22100,
-    LHW: 22200,
-    ZHY: 22300,
-    SJW: 22400,
-    PKX: 22500,
-    BJS: 22600,
-    TSN: 22700,
-    TNA: 22800,
-    TAO: 22900,
-    HAK: 23000,
-    FUO: 23100,
-    ZGN: 23200,
-    CAN: 23300,
-    SZX: 23400,
-    MDL: 23500,
-    RGN: 23600,
-    CNX: 23700,
-    BKK: 23800,
-    URT: 23900,
-    KUL: 24000,
-    JHB: 24100,
-    QPG: 24200,
-    SIN: 24300,
-    CGK: 24400,
-    JOG: 24500,
-    DPS: 24600,
-    DEX: 24700,
-    PER: 24800,
-    ADL: 24900,
-    MEL: 25000,
-    HBA: 25100,
-    CBR: 25200,
-    SYD: 25300,
-    BNE: 25400,
-    CHC: 25500,
-    WLG: 25600,
-    AKL: 25700,
-    NOU: 25800,
-    SUV: 25900,
-    PPT: 26000,
-    GDL: 26100,
-    QRO: 26200,
-    MEX: 26300,
-    GUA: 26400,
-    TGU: 26500,
-    SJO: 26600,
-    PTY: 26700,
-    MDE: 26800,
-    BOG: 26900,
-    UIO: 27000,
-    GYE: 27100,
-    LIM: 27200,
-    ARI: 27300,
-    SCL: 27400,
-    COR: 27500,
-    NQN: 27600,
-    EZE: 27700,
-    ASU: 27800,
-    CGB: 27900,
-    GYN: 28000,
-    BSB: 28100,
-    UDI: 28200,
-    SJP: 28300,
-    RAO: 28400,
-    VCP: 28500,
-    SOD: 28600,
-    GRU: 28700,
-    QWJ: 28800,
-    SJK: 28900,
-    GIG: 29000,
-    CNF: 29100,
-    CAW: 29200,
-    VIX: 29300,
-    SSA: 29400,
-    REC: 29500,
-    JDO: 29600,
-    FOR: 29700,
-    BEL: 29800,
-    MAO: 29900,
-    POA: 30000,
-    FLN: 30100,
-    ITJ: 30200,
-    NVT: 30300,
-    BNU: 30400,
-    JOI: 30500,
-    CWB: 30600,
-    CFC: 30700,
-    XAP: 30800,
-    PBM: 30900,
-    GEO: 31000,
-    BGI: 31100,
-    GND: 31200,
-    CUR: 31300,
-    SDQ: 31400,
-    STI: 31500,
-    KIN: 31600,
-    MIA: 31700,
-    TPA: 31800,
-    GNV: 31900,
-    JAX: 32000,
-    TLH: 32100,
-    MGM: 32200,
-    ATL: 32300,
-    PDK: 32400,
-    BNA: 32500,
-    MEM: 32600,
-    STL: 32700,
-    IND: 32800,
-    CHI: 32900,
-    ORD: 33000,
-    DTW: 33100,
-    CLE: 33200,
-    PIT: 33300,
-    CMH: 33400,
-    LCK: 33500,
-    CLT: 33600,
-    RDU: 33700,
-    ORF: 33800,
-    RIC: 33900,
-    IAD: 34000,
-    DCA: 34100,
-    PHL: 34200,
-    NYC: 34300,
+export const airportCodeToRegionNumber: Record<string, [regionNumber: number, latitude: number, longitude: number]> = {
```

Comment:
Could rename the function since it now also returns coordinates.

Reply:
Could also return the value in a map instead of an array to make the code that uses the function more understandable.

Reply:
This is actually generated data (not a function), and there are a lot of code that depends on the structure of this data (the data gets sorted etc). The file is large already, and converting it into objects would almost double its size. I think it is better not to touch this.